portal/public/js/mojFrontend.js.LICENSE.txt

```diff
@@ -1,10 +1,10 @@
 /*!
- * jQuery JavaScript Library v3.7.0
+ * jQuery JavaScript Library v3.7.1
  * https://jquery.com/
  *
  * Copyright OpenJS Foundation and other contributors
  * Released under the MIT license
  * https://jquery.org/license
  *
- * Date: 2023-05-11T18:29Z
+ * Date: 2023-08-28T13:37Z
  */
```

Comment:
Why did this and `portal/public/js/mojFrontend.js.map` change?

Reply:
@oscar-richardson-softwire  and I looked into this: 2 months ago we updated the version of jquery in portal from 3.7.0 to 3.7.1 but we have not done a webpack build since then